target/linux/ramips/dts/mt7620a_apexera_wibot.dts

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera,wibot", "ralink,mt7620a-soc";
+	model = "Apexera WiBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+	};
+
+	aliases {
+		led-boot = &led_boot;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		3g {
+			label = "wibot:white:3g";
+			gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		};
+
+		wifi {
+			label = "wibot:white:wifi";
+			gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
```

Comment:
No trigger?

Reply:
I have checked the dts referring to mt7620a, as for wifi led, no trigger found for it. do you have any suggestion ?

Reply:
And it doesn't work if you just put "linux,default-trigger = "phy0tpt";" there?